README.md

```diff
@@ -28,11 +30,14 @@ Setup the application:
 
     $ cp development.ini.sample development.ini
     $ gearbox setup-app
+If `gearbox setup-app` fails try specifying `pip3` in the previos two steps
 
-Finally, serve the application:
+Serve the application:
 
     $ gearbox serve --reload --debug
 
+Finaly, go into your web browser and go to localhost:8080 to view the website.
```

Comment:
1. "... go ***to*** your web browser and **navigate** to ..."
2. Wrap "localhost:8080" in backticks to make it render as monospace:

        ... to `localhost:8080` to ...